opencl/opencl-aot/source/main.cpp

```diff
@@ -322,10 +322,29 @@ int main(int Argc, char *Argv[]) {
                         cl::desc("Input OpenCL program binary files"));
   cl::alias OptIR("ir", cl::aliasopt(OptOutputElf), cl::desc("Output file"));
 
-  enum ArchType : int8_t { sse42 = 0, avx, avx2, avx512 };
+  enum ArchType : int8_t {
+    sse42 = 0,
+    avx,
+    avx2,
+    avx512,
+    wsm,
+    snb,
+    ivyb,
+    bdw,
+    cfl,
+    adl,
+    skylake,
+    skx,
+    clk,
+    icl,
+    icx,
+    spr
+  };
   cl::opt<ArchType> OptMArch(
-      "march",
-      cl::desc("Set target CPU architecture (for --device=cpu or "
+      "March",
```

Comment:
march is used in LLVM CodeGen https://github.com/intel/llvm/blob/sycl/llvm/lib/CodeGen/CommandFlags.cpp#L115.

Reply:
Hi @haonanya 

I have similar concerns. Can you please provide a tool flow where this tool is built in conjunction with LLVM codegen? Also, I see both the options have existed for atleast three years now. I am curious to see why did the issue crop up just now?

Thanks

Reply:
Thanks, I will continue to investigate it deeper.